fields/types/date/test/type.js

```diff
@@ -67,14 +67,14 @@ exports.testFieldType = function (List) {
 			});
 		});
 
-		it('should clear the value when passed undefined', function (done) {
+		it('should not clear the value when a value exists and passed undefined', function (done) {
 			var testItem = new List.model({
 				date: '2015-01-01',
 			});
 			List.fields.date.updateItem(testItem, {
 				date: undefined,
 			}, function () {
-				demand(testItem.date).be(null);
+				demand(testItem.date).not.be.null();
```

Comment:
This test was a bit weird, to stay consistent with the other fields I've now changed it and fixed the implementation – was this intended behaviour?